rfcs/0001-rfc-process.md

```diff
@@ -0,0 +1,135 @@
+
+## What is an RFC?
+
+RFC refers to the IETF's [Request For Comments](https://en.wikipedia.org/wiki/Request_for_Comments) process. An RFC is a document or set of documents that describe a feature in detail, and go through a process of review and approval.
+
+For Lit, RFCs are markdown files in the lit/rfcs repository.
+
+RFCs have a structure as defined in the RFC template (_TODO_), including these metadata and sections (the RCF template will include more descriptions):
+
+* Champion(s): The people making and maintaining the proposal. This can change over time.
+* Stage: Accepted, Deprecated, Implemented
+* Summary
+* Objective (goals and non-goals)
+* Motivation (background)
+* Detailed Design, including these recommended sections:
+    * Implementation Plan
+    * Backward Compatibility
+    * Testing Plan
+    * Performance and Code Size Impact
+    * Interoperability
+    * Security Impact
+    * Documentation Plan
+* Downsides
+* Alternatives
+
+Accepted RFCs are not a guarantee that a feature will be developed and included in Lit. They are an indication that the team has consensus on a feature and design, and is open to accepting PRs to implement it. Things can change, and new information such as implementation feedback may lead the team to deprecate an RFC.
+
+## When to use RFCs
+
+RFCs must be created for new and "substantial" changes. "Substantial" is somewhat subjective, but some clear signs are:
+
+* New packages
+* New modules, classes, directives, etc. in an existing package
+* Most new public-facing API
+* Breaking changes
+
+Changes that do _not_ require an RFC:
+
+* Internal implementation changes
+* Bug fixes
+* Most dev mode changes: warnings, etc.
+
+These criteria leave a category of small changes and experiments open to debate on whether they deserve or require an RFC. We do not wish to slow down development with too much process, so we will leave this area to the discretion of the core team. We will adjust the criteria for requiring RCFs over time.
+
+#### Relationship to Lit Labs
+
+The Lit project also has a labs process for pre-production experimentation and feedback. With the assumption that when we create a labs package we usually intend for it to eventually graduate, most labs packages should have (or be part of) an RFC.
+
+## RFC Process
+
+### Before opening an RFC
+
+RFCs involve more work than an issue, both to open one and to review one. Before opening an RFC is it a good idea to get some indication of interest from the Lit core team or strong support from the community.
+
+You may get this indication in several ways, from informal to formal:
+
+* A feature request issue is closed with a request to open an RFC
+* A core team member directly invites you to open an RFC from a community or support channel like GitHub Discussions, Slack, or Twitter
+* You open a "RRFC" (Requesting Request For Comment) issue in lit/rfcs and get positive feedback 
```

Comment:
It would be nice to make some templates like we have for https://github.com/lit/lit/issues/new/choose for RRFCs.

I guess the other kind of issue would be for comments/questions/proposals about already merged RFCs?

Reply:
Yeah, those two categories make sense to me. I think the only other would be meta about the process itself.